gradle.properties

```diff
@@ -34,3 +34,6 @@ junit_version=4.12
 lombok_version = 1.16.12
 #
 hibernate.version=5.0.2.Final
+#
+keycloak_spring_boot_version=3.0.0.Final
```

Comment:
is this the same version as the tomcat8 adapter by chance, or are the two versions always the same? If they are always the same, please use the variable `keycloak_spring_boot_version` for both.